internal/context/workspace/domain/workflow/reader.go

```diff
@@ -0,0 +1,164 @@
+package workflow
+
+import (
+	"bufio"
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"github.com/Bio-OS/bioos/internal/context/workspace/interface/grpc/proto"
+	apperrors "github.com/Bio-OS/bioos/pkg/errors"
+	applog "github.com/Bio-OS/bioos/pkg/log"
+	"github.com/Bio-OS/bioos/pkg/utils/exec"
+	"os"
+	"path"
+	"path/filepath"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+// ReaderOptions workflow reader options
+type ReaderOptions struct {
+	WomtoolPath string
```

Comment:
This is targeting at wdl only.

Reply:
let me think about how to make ReaderOptions more suitable.